src/app/pages/profile/profile.component.html

```diff
@@ -0,0 +1,113 @@
+<!-- app.component.html -->
+<mat-toolbar color="primary">
+  <span>Profile</span>
+</mat-toolbar>
+<div>
+  <mat-card class="profile-card">
+    <div class="coverImageSection">
+      <button class="addCoverImageButton">Add cover image</button>
+    </div>
+    <mat-card-header>
+      <div mat-card-avatar class="profile-picture">
+        <mat-icon
+          aria-hidden="false"
+          aria-label="Example home icon"
+          fontIcon=""
+          role="img"
+          style="width: 200px; height: 200px; font-size: 100px"
+          >account_circle</mat-icon
+        >
+      </div>
+      <div style="display: flex; align-items: end; margin-left: 20px">
+        <mat-card-title *ngIf="user !== null">{{
+          user.firstName + " " + user.lastName
+        }}</mat-card-title>
+      </div>
+    </mat-card-header>
+  </mat-card>
+  <div class="heroSection">
+    <div class="leftArea"></div>
+    <div class="postsArea">
+      <h2 style="display: flex">Recent posts</h2>
+      <mat-form-field
+        class="example-full-width"
+        style="width: 42vw; height: 200px"
+      >
+        <mat-label>What's on your mind?</mat-label>
+        <input matInput placeholder="Ex. Pizza" value="" />
+      </mat-form-field>
+      <button class="addPostButton">Post</button>
+      <div *ngFor="let post of posts">
+        <mat-card-content class="postCard">
+          <div class="d-flex">
+            <mat-icon
+              aria-hidden="false"
+              aria-label="Avatar"
+              role="img"
+              style="font-size: 40px; width: 45px; height: 50px"
+              >account_circle
+            </mat-icon>
+            <div style="display: flex; flex-direction: column">
+              <span style="font-weight: 800" *ngIf="user !== null">
+                {{ user!.firstName + " " + user.lastName }}
```

Comment:
```suggestion
                {{ user?.firstName + " " + user.lastName }}
```
Cum e acum o sa dea crash daca e undefined... cu `?` macar doar apare string gol

Reply:
Da resolve la conversatie daca decizi sa nu implementezi asta :))

Reply:
Am considerat a se merge doar pe un happy flow fara validari